### ISSUE TITLE
Assertion violation at src/tsolvers/lasolver/Simplex.cc:460

### ISSUE THREAD

**Issue by rainoftime** [usi-verification-and-security/opensmt#71]
Hi, for the following instance
~~~~
(set-logic QF_LIA) 
(declare-fun v0 () Bool)
(declare-fun v2 () Bool)
(declare-fun v4 () Bool)
(declare-fun v5 () Bool)
(declare-fun i4 () Int)
(declare-fun i7 () Int)
(declare-fun i12 () Int)
(declare-fun v7 () Bool)
(push 1)
(assert (= (=> (= v4 (> i7 i4)) (and v2 (= v0 v0))) (= v2 v7)))
(check-sat)
(push 1)
(assert (= (> (+ i7 811) 811) v5))
(pop 1)
(pop 1)
(assert (> i7 i12))
(push 1)
(check-sat)
~~~~

opensmt (commit e9e14b4, built with ASAN) throws an assertion violation
~~~~
opensmt: /opensmt/src/tsolvers/lasolver/Simplex.cc:460: opensmt::Real Simplex::computeDelta() const: Assertion `!isModelOutOfBounds(v)' failed.
timeout: the monitored command dumped core
Aborted
~~~~

**Comment by aehyvari** [comment]
Hi Jack,

I'm wondering how critical these reports are for your current use case.
We're developing a lot of features right now related, for instance, to
incrementality.  If your issues are not arising from a specific urgent
need, it might make sense to wait for the new version to come out.  It will
have its own unique problems that are probably completely different from
those of the current version, and therefore we'd anyway need a new set of
regression tests for those.

In any case thanks for reporting these!

Cheers,
Antti

**Comment by rainoftime** [comment]
@aehyvari  Thanks for the replay! Hope the test cases could help

**Comment by blishko** [comment]
Seems that this has also been fixed by #80
